cli/util/cancel.go

```diff
@@ -52,7 +52,15 @@ func cancelTx(ctx *cli.Context) error {
 
 	mainTx, _ := c.GetRawTransactionVerbose(txHash)
 	if mainTx != nil && !mainTx.Blockhash.Equals(util.Uint256{}) {
-		return cli.NewExitError(fmt.Errorf("transaction %s is already accepted at block %s", txHash, mainTx.Blockhash.StringLE()), 1)
```

Comment:
To me this is a correct behavior because we can't satisfy `canceltx` request. Test can get it, true, but likely it can be adjusted to handle this case.

Reply:
The aim of this PR is unification, because right now we have two cases when target tx is accepted (before conflict construction and after conflict is sent to mempool), and both should return the same code.

So it's OK to me, but then how about the case when neither target nor conflicting tx is accepted and the node doesn't know VUB of target tx? I thought that *this* case is a command error.

Reply:
Fixed.